.vib/odoo/cypress/cypress/integration/odoo_spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+import 'cypress-real-events/support';
+
+it('allows creating a company', () => {
+  cy.login();
+  cy.get('[title="Home Menu"]').click();
+  cy.contains('Settings').click();
+  cy.contains('Users & Companies').click();
+  cy.get('[data-menu-xmlid="base.menu_action_res_company_form"]').click();
+  cy.get('.o_list_button_add').click();
+  cy.fixture('companies').then((company) => {
+    cy.get('[name="name"]').type(`${company.newCompany.name}.${random}`);
+    cy.contains('Save').click();
+    cy.contains('.oe_title', 'Company Name');
+    cy.contains('.oe_title', `${company.newCompany.name}.${random}`);
```

Comment:
Isn't this duplicated?

Reply:
No, I want to check for both texts in the same element. It is on purpose.